buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -906,11 +906,12 @@ public static ByteBuf threadLocalDirectBuffer() {
         }
 
         if (buf.hasArray()) {
-            if (copy || start != 0 || length != capacity) {
-                int baseOffset = buf.arrayOffset() + start;
-                return Arrays.copyOfRange(buf.array(), baseOffset, baseOffset + length);
+            int baseOffset = buf.arrayOffset() + start;
+            byte[] v = buf.array();
```

Comment:
How about calling this `bytes` instead of `v`?

Reply:
I did that for consistency with `byte[] v = PlatformDependent.allocateUninitializedArray(length);` later in the same method (it's _just_ cut off by GitHub's diff viewer), but sure. I could change both so they're still consistent?

Reply:
`bytes` or `array` is a good name IMO. I'm not sure what `v` means.

See https://github.com/netty/netty/blob/a76580bbc673561281613964b65eb8381bc2febb/buffer/src/main/java/io/netty/buffer/ByteBufUtil.java#L844

Reply:
I've renamed it to `bytes`, along with the existing `byte[] v` in that method (in a separate commit).